views/js/qtiCreator/editor/customElementRegistryFactory.js

```diff
@@ -45,7 +45,9 @@ define([
             var paths = {};
 
             _(hooks).values().each(function(hook){
-
+                if (hook.manifest && hook.manifest.disabled) {
+                    return;
+                }
                 if(isValidHook(hook)){
```

Comment:
Seems a bit extreme to me, but it does the job for now. I would have suggested to still enable it to be registered and let the hook decide how they should handle disabled custom elements (it may be hidden, it may be grayed out, etc.) With this solution, it is as if they do not exist. But don't worry, we can leave it for now and change it if we need another behaviour.